developers/weaviate/current/client-libraries/python.md

```diff
@@ -524,8 +554,9 @@ The `Batch` object can be configured using the `batch.configure()` method or the
 - `dynamic` - (`bool`: default: `False`): Enables/disables Dynamic-batching. Does not have any effect if `batch_size` is `None`.
 - `creation_time` - (`int` or `float`; default: `10`): It is the interval of time in which the batch import/create should be done. It used to compute `recommended_num_objects` and `recommended_num_references`, consequently has an impact for Dynamic-batching.
 - `callback` (Optional[Callable[[dict], None]]: default `weaviate.util.check_batch_result`): It is a callback function on the results of the `batch.create_objects()` and `batch.create_references()`. It is used for Error Handling for Auto-/Dynamic-batching. Has no effect if `batch_size` is `None`.
-- `timeout_retries` - (`int`: default `0`): Number of times to re-try to import/create a batch that resulted in `TimeoutError`.
-
+- `timeout_retries` - (`int`: default `3`): Number of re-try to do on import/create a batch that resulted in `TimeoutError`.
+- `connection_error_retries` - (`int`: default `3`): Number of re-try to do on import/create a batch that resulted in `ConnectionError`. **NOTE:** Available in `weaviate-client>=3.9.0`.
+- `num_workers` - (`int`: default `0`): The maximal number of concurrent threads to run batch import. Only used for non-MANUAL batching. i.e. is used only with AUTO or DYNAMIC batching. ***Use with care to not overload your weaviate instance.*** **NOTE:** Available in `weaviate-client>=3.9.0`.
```

Comment:
default is 1

Reply:
Oooo, my bad. It cannot actually be set to 0 )))